<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task2</class>
 <widget class="QWidget" name="task2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="array">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>enter array</string>
   </property>
  </widget>
  <widget class="QComboBox" name="option_oe">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="matrix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="funcArr">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Process array</string>
   </property>
  </widget>
  <widget class="QPushButton" name="funcMat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Process matrix</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
